docs/section10/10-7.md

```diff
@@ -29,48 +29,45 @@
+                        - Set the full load no fan efficiency: `full_load_efficiency_b = cooling_system_b.efficiency_metric_values[x]`
+                        - Set the efficiency found boolean variable to TRUE (if never found outcome to RDS is fail): `was_efficiency_found = TRUE`
+                    - Else, do nothing and move on to the next metric: `Else:`          
+                Based upon the system type and cooling capacity determine the efficiency that is expected to be modeled.  
+                - Check if the system is system type 1 or 1b: `if hvac_sys_type_b = "SYS-1" or hvac_sys_type_b = "SYS-1b":`   
                     - Set COPnf_full_load_eff_b = 3.2:`COPnf_full_load_eff_b = 3.2`
                     - Check if the modeled efficiency is consistent with this value, if so set boolean variable to true: `if full_load_efficiency_b == COPnf_full_load_eff_b: meets_lower_or_equiv_size_category_eff_b = TRUE`
                 - Check else if the system is system type 2: `elif hvac_sys_type_b = "SYS-2":`
-                    - Set COPnf_full_load_eff_b = 3.2:`COPnf_full_load_eff_b = 3.1`
+                    - Set COPnf_full_load_eff_b = 3.1:`COPnf_full_load_eff_b = 3.1`
                     - Check if the modeled efficiency is consistent with this value, if so set boolean variable to true: `if full_load_efficiency_b == COPnf_full_load_eff_b: meets_lower_or_equiv_size_category_eff_b = TRUE`
-                - Check else if the system type is 3, 3b and that num_zones_defined_for_all_zones equals False (pass if efficiency meets a lower size category or the size category associated with the system):`elif (hvac_sys_type_b = "SYS-3" or hvac_sys_type_b = "SYS-3b") AND num_zones_defined_for_all_zones == FALSE:`
+                - Check else if the system type is 3, 3b and that the associated zone.equipment_count equal Null, this means the number of aggregated hvac zones was not defined (pass if efficiency meets a lower size category or the size category associated with the system). Assumes that the HVAC system serves one zone which is consistent with the rules of App G.:`elif (hvac_sys_type_b = "SYS-3" or hvac_sys_type_b = "SYS-3b") AND zone_list_b[0].equipment_count == Null:`
                     - Get baseline efficiency for the <65,000 Btuh size category:`COPnf_full_load_65Btuh_b = data_lookup(table_G3_5_1,65Btuh)`
                     - Check if the modeled efficiency is consistent with this value, if so set boolean variable to true: `if full_load_efficiency_b == COPnf_full_load_65Btuh_b: meets_lower_or_equiv_size_category_eff_b = TRUE`
-                    - Check if capacity is not in the <65,000 Btuh size category: `if if total_cool_capacity_b >= 65000:`
+                    - Check if capacity is greater or equal to 65000 and the efficiency is not equal to a lower size bracket efficiency: `if total_cool_capacity_b >= 65000 and meets_lower_or_equiv_size_category_eff_b != TRUE:`
```

Comment:
As mentioned above, efficiency for capacity bracker over 760 kBtu/Hr is higer than for 65 Kbtu/Hr

Reply:
Per email, implemented logic such that if the aggregation_factor (new name) is undefined and the efficiency is consistent with the size category assuming no zone aggregation OR is a more conservative efficiency from a lower bracket and size breacket we pass it